specification/cognitiveservices/data-plane/WebSearch/v1.0/WebSearch.json

```diff
@@ -0,0 +1,1211 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Web Search API",
+    "description": "The Web Search API lets you send a search query to Bing and get back search results that include links to webpages, images, and more.",
+    "version": "1.0"
+  },
+  "host": "api.cognitive.microsoft.com",
+  "schemes": [
+    "https"
+  ],
+  "basePath": "/bing/v7.0",
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "apiKeyHeader": {
+      "name": "Ocp-Apim-Subscription-Key",
+      "type": "apiKey",
+      "in": "header"
+    },
+    "apiKeyQuery": {
+      "name": "subscription-key",
+      "type": "apiKey",
+      "in": "query"
+    }
+  },
+  "security": [
+    {
+      "apiKeyHeader": []
+    },
+    {
+      "apiKeyQuery": []
+    }
+  ],
+  "paths": {
+    "/search": {
+      "get": {
+        "summary": "The Web Search API lets you send a search query to Bing and get back search results that include links to webpages, images, and more.",
+        "operationId": "Web_Search",
+        "tags": [
+          "WebSearch"
+        ],
+        "parameters": [
+          { "$ref": "#/parameters/x-bingapis-sdk" },
+          {
+            "name": "Accept",
```

Comment:
I don't think we need to specify the `Accept` header -- do we? This just complicates the API and we really can't change the generated code to do anything but `application/json`

Reply:
Not sure, it was definitely called out in our public documentation for the cognitive API though (at least for EntitySearch)

Reply:
The `produces` property in swagger tells a client what content types the service supports (and thus implies the what set of valid values for the Accept header is for a given endpoint). I would suggest using that to accomplish your goals.

Reply:
Public documentation for Web Search API 
https://docs.microsoft.com/en-us/rest/api/cognitiveservices/bing-web-api-v7-reference